src/Core/Utils.fs

```diff
@@ -194,18 +212,15 @@ module Markdown =
         comment
         |> String.replace "&lt;" "<"
         |> String.replace "&gt;" ">"
-        |> (fun v -> v.Split '\n')
-        |> Array.filter(String.IsNullOrWhiteSpace>>not)
-        |> Array.mapi (fun i line ->
-            let v =
-                if i = 0 && not (String.IsNullOrWhiteSpace line)
-                then "\n" + line.Trim()
-                else line.Trim()
-            replaceXml v
-            )
+        |> String.replace "&quot;" "\""
+        |> String.replace "&apos;" "'"
+        |> String.replace "&amp;" "&"
+        |> replaceXml
+        |> String.split [|'\n'|]
+        |> Array.filter (not << String.IsNullOrWhiteSpace)
+        |> Array.map String.trim
         |> String.concat "\n\n"
-        |> String.trim
-        |> (fun v -> MarkdownString (v))
+        |> (fun v -> MarkdownString v)
```

Comment:
This can be replaced with just `|> MarkdownString`

Reply:
I tried that, but the compiler complains with `This expression was expected to have type 'string' but here has type 'unit'`. Maybe because the parameter to MarkdownString is optional?

Reply:
Yep, I confirmed it, that syntactic sugar doesn't work when object constructors have an optional parameter